lib/main.dart

```diff
@@ -86,3 +180,57 @@ class _MyHomePageState extends State<MyHomePage> {
     );
   }
 }
+
+class Storage {
```

Comment:
Might be good practice to move this to it's own 'storage' file and import it

Reply:
Did it, let me know what you think.